### PR TITLE
Added PVC deletion check in test-storage-pool and openebs-percona-ownnamespace playbook.

### PR DESCRIPTION
**Pull request by Swarnalatha-K**
**What this PR does / why we need it**:
Added PVC deletion check and slack Notification in deploy openebs and percona its ownnamespace playbook and test-storage-pool playbook.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
@ksatchit please review this PR.

### CODE REVIEW

File: e2e/ansible/playbooks/hyperconverged/percona-openebs-ownnamespace/k8s-percona-openebs-pod-cleanup.yml [hunk 1]
```diff
@@ -1,35 +1,51 @@
...
+         shell: source ~/.profile; kubectl get pods --all-namespaces -l name=percona
+         args:
+           executable: /bin/bash
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+         register: result
+         until: "'percona' not in result.stdout"
+         delay: 30
+         retries: 10
+
+       - name: Get pvc name to verify successful pvc deletion
```
Comment (dargasudarshan) [inline e2e/ansible/playbooks/hyperconverged/percona-openebs-ownnamespace/k8s-percona-openebs-pod-cleanup.yml]: Can you get pvc name before initiating application delete?

File: e2e/ansible/playbooks/hyperconverged/test-k8s-storage-pool/cleanup.yml [hunk 2]
```diff
@@ -14,6 +14,24 @@
...
          retries: 6
+
+       - name: Get pvc name to verify successfull pvc deletion 
+         shell: source ~/.profile; kubectl get pvc --all-namespaces | grep {{ volume_claim }} |  awk {'print $3'}
+         args:
+         executable: /bin/bash
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+         register: pvc
+
+       - name: Confirm pvc pod has been deleted
```
Comment (dargasudarshan) [inline e2e/ansible/playbooks/hyperconverged/test-k8s-storage-pool/cleanup.yml]: Same as previous comment.

**Comment by Swarnalatha-K** [comment]
```
+ openebs maya-apiserver-6d945596b-rp4ck › maya-apiserver
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver + MAYA_API_SERVER_NETWORK=eth0
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver + ip -4 addr show scope global dev eth0
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver + grep inet
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver + awk {print $2}
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver + cut -d / -f 1
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver + CONTAINER_IP_ADDR=10.36.0.1
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver + exec /usr/local/bin/maya-apiserver start --bind=10.36.0.1
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:25:33.109898       6 start.go:122] Initializing maya-apiserver...
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:25:33.110437       6 start.go:231] Starting maya api server ...
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:25:33.110482       6 registry.go:50] Registered 'kubernetes' as orchestration provider
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:25:33.110521       6 registry.go:50] Registered 'nomad' as orchestration provider
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:25:33.115626       6 start.go:160] Maya api server configuration:
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:25:33.115697       6 start.go:162]          Log Level: INFO
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:25:33.115708       6 start.go:162]             Region: global (DC: dc1)
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:25:33.115736       6 start.go:162]            Version: v0.6.0-unreleased
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:25:33.115742       6 start.go:168]
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:25:33.115747       6 start.go:171] Maya api server started! Log data will stream in below:
+ openebs openebs-provisioner-7d546d4cd9-ph9hh › openebs-provisioner
openebs openebs-provisioner-7d546d4cd9-ph9hh openebs-provisioner I0425 09:25:33.940799       1 maya_api.go:63] OpenEBS volume provisioner namespace openebs
openebs openebs-provisioner-7d546d4cd9-ph9hh openebs-provisioner I0425 09:25:34.220603       1 controller.go:407] Starting provisioner controller 9b0faecc-486a-11e8-a845-2af9bb302963!
openebs openebs-provisioner-7d546d4cd9-ph9hh openebs-provisioner I0425 09:29:53.870017       1 controller.go:1080] scheduleOperation[lock-provision-percona/percona-diffns-claim[35f1f67b-486b-11e8-ae05-02b983f0a4db]]
openebs openebs-provisioner-7d546d4cd9-ph9hh openebs-provisioner I0425 09:29:53.927207       1 controller.go:1080] scheduleOperation[lock-provision-percona/percona-diffns-claim[35f1f67b-486b-11e8-ae05-02b983f0a4db]]
openebs openebs-provisioner-7d546d4cd9-ph9hh openebs-provisioner I0425 09:29:54.096925       1 leaderelection.go:156] attempting to acquire leader lease...
openebs openebs-provisioner-7d546d4cd9-ph9hh openebs-provisioner I0425 09:29:54.226336       1 leaderelection.go:178] successfully acquired lease to provision for pvc percona/percona-diffns-claim
openebs openebs-provisioner-7d546d4cd9-ph9hh openebs-provisioner I0425 09:29:54.226545       1 controller.go:1080] scheduleOperation[provision-percona/percona-diffns-claim[35f1f67b-486b-11e8-ae05-02b983f0a4db]]
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver 2018/04/25 09:29:54.298088 [DEBUG] http: Request /latest/volumes/ (POST)
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver [DEBUG] Processing POST request
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:29:54.298143       6 volume_endpoint.go:263] Processing Volume add request
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:29:54.488158       6 util.go:1219] Will use env var 'DEFAULT_PVP_NAME: '
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:29:54.488432       6 jiva.go:56] Building new instance of jiva persistent volume provisioner 'volumeprovisioner.mapi.openebs.io/name:jiva'
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:29:54.489697       6 util.go:1219] Will use env var 'DEFAULT_ORCHESTRATOR_NAME: '
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:29:54.489770       6 k8s.go:59] Building 'orchprovider.mapi.openebs.io/name':'kubernetes' orchestration provider
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:29:54.489874       6 util.go:1219] Will use env var 'DEFAULT_ORCHESTRATOR_IN_CLUSTER: '
openebs maya-apiserver-6d945596b-rp4ck maya-apiserver I0425 09:29:54.497336       6 k8s.go:1180] Adding service for Volume 'name : pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db'
+ percona pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-rep-78759d8b8-pgfv6 › pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-rep-con
percona pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-rep-78759d8b8-pgfv6 pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-rep-con time="2018-04-25T09:29:59Z" level=info msg="Creating volume /openebs, size 5368709120/512"
percona pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-rep-78759d8b8-pgfv6 pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-rep-con time="2018-04-25T09:29:59Z" level=info msg="Listening on data 10.39.0.3:9503"
percona pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-rep-78759d8b8-pgfv6 pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-rep-con time="2018-04-25T09:30:00Z" level=info msg="Listening on control 10.39.0.3:9502"
percona pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-rep-78759d8b8-pgfv6 pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-rep-con time="2018-04-25T09:30:00Z" level=info msg="Listening on sync 10.39.0.3:9504"
+ percona pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-ctrl-79767b4fb6-7c5f4 › maya-volume-exporter
+ percona pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-ctrl-79767b4fb6-7c5f4 › pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-ctrl-con
percona pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-ctrl-79767b4fb6-7c5f4 maya-volume-exporter I0425 09:30:10.857049       1 command.go:70] Starting maya-volume-exporter ...
percona pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-ctrl-79767b4fb6-7c5f4 maya-volume-exporter I0425 09:30:10.857281       1 logs.go:43] Starting Server: :9500
percona pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-ctrl-79767b4fb6-7c5f4 pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-ctrl-con mount: special device /host/dev does not exist
percona pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-ctrl-79767b4fb6-7c5f4 pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-ctrl-con time="2018-04-25T09:29:59Z" level=info msg="Listening on :9501"
percona pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-rep-78759d8b8-hpp57 pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-rep-con time="2018-04-25T09:30:13Z" level=info msg="Removing Replica"
percona pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-ctrl-79767b4fb6-7c5f4 pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-ctrl-con STATE = closed
percona pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-ctrl-79767b4fb6-7c5f4 pvc-35f1f67b-486b-11e8-ae05-02b983f0a4db-ctrl-con 10.36.0.0 - - [25/Apr/2018:09:30:13 +0000] "POST /v1/register HTTP/1.1" 200 0
@
```

**Comment by Swarnalatha-K** [comment]
@yudaykiran Added stern steps and output please review it.

**Comment by yudaykiran** [comment]
@Swarnalatha-K , just as a final check can you post the ansible-lint output for the test playbooks. If it throws some linting errors put them here and also put the resolution used to solve those linting errors.

**Comment by Swarnalatha-K** [comment]
```ansible-lint output:
devops@devops-virtual-machine:~/openebs/e2e/ansible/playbooks/hyperconverged/test-k8s-storage-pool$ ansible-lint test-storage-pool.yml
devops@devops-virtual-machine:~/openebs/e2e/ansible/playbooks/hyperconverged/test-k8s-storage-pool$ ```

**Comment by Swarnalatha-K** [comment]
@yudaykiran @ksatchit Resolved ansible-lint errors. can you please review it.

**Pull request merged**